src/freertos_drivers/ti/TivaDCC.hxx

```diff
@@ -274,7 +274,14 @@ public:
 
     /// Turns on DCC output.
     void enable_output() {
```

Comment:
You could choose to fix the brace to a hanging brace.  Not technically part of the pull request.  Up to you.

Reply:
i'll rather reformat the whole file in one go.